### PR TITLE
Add ISO-image implementation & minor docs fixes

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

File: gridscale/datasource_gridscale_isoimage_test.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		},
+	})
+
+}
+
+func testAccCheckDataSourceGridscaleISOImageConfig_basic(name string) string {
+	return fmt.Sprintf(`
+resource "gridscale_isoimage" "foo" {
+  name   = "%s"
+  source_url = "http://tinycorelinux.net/10.x/x86/release/TinyCore-current.iso"
```
Comment (bkircher) [inline gridscale/datasource_gridscale_isoimage_test.go:35]: :heart:

File: website/docs/d/isoimage.html.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+layout: "gridscale"
+page_title: "gridscale: ISO-Image"
```
Comment (bkircher) [inline website/docs/d/isoimage.html.md]: Please change here, too.

**Review by bkircher — APPROVED** [review]
LGTM. Please fix a couple of typos.

**Review by itakouna — COMMENTED** [review]
@nvthongswansea Thanks for fixing `gridscale_template_test`.  Plz, only fix the comments from @bkircher. LGTM

File: gridscale/datasource_gridscale_isoimage_test.go (hunk 1 above)
Comment (nvthongswansea) [reply gridscale/datasource_gridscale_isoimage_test.go:35]: What is wrong with this one please?

File: gridscale/datasource_gridscale_isoimage.go [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+			server := map[string]interface{}{
+				"object_uuid": value.ObjectUUID,
+				"create_time": value.CreateTime.String(),
+				"object_name": value.ObjectName,
+				"bootdevice":  value.Bootdevice,
+			}
+			servers = append(servers, server)
+		}
+		log.Printf("aaa %v", props)
+		log.Printf("bbb %v", servers)
```
[the remark this answers is not in the document]
Comment (nvthongswansea) [reply gridscale/datasource_gridscale_isoimage.go:171]: I already removed them in another branch :D

File: gridscale/datasource_gridscale_isoimage_test.go (hunk 1 above)
Comment (bkircher) [reply gridscale/datasource_gridscale_isoimage_test.go:35]: Nothing :slightly_smiling_face: Just happy to see tiny core linux mentioned again

**Comment by bkircher** [comment]
Thanks @nvthongswansea

**Pull request merged**